docs/contributors/git workflow tips.md

```diff
@@ -3,7 +3,7 @@ GTG uses [Git](https://git-scm.com) for versioning. It might be useful to take a
 
 # Getting the code
 
-Get the latest version of the GTG code [on GitHub](https://github.com/getting-things-gnome/gtg). We suggest forking the master branch at first (you can do this with the GitHub web interface). Then clone the forked repository from GitHub to your local computer:
+Get the latest version of the GTG code on GitHub. We suggest forking the master branch at first (you can do this with the GitHub web interface). Then clone the forked repository from GitHub to your local computer:
```

Comment:
Why remove this link?

Reply:
I didn't mean to do that, but I was sleepy.

Reply:
No prob. I think the last commit you added (a9625c6) re-adds this change

Reply:
It would be better to use `git rebase -i` and apply your correction commit as a "fixup" to where it is needed, instead of an extra commit.